### PR TITLE
Bug fixed for .dockerignore

### PR DESCRIPTION
**Pull request by lunny**
Gitea will be COPY to container, so execluded ignore file.

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 *
+!gitea
 !docker
 !bin
```
Comment (tboerger) [inline .dockerignore]: Also drop this please, or just replace this.

Comment (lunny) [reply .dockerignore]: Done. :)

**Comment by tboerger** [comment]
LGTM

**Comment by thibaultmeyer** [comment]
LGTM

**Pull request merged**